cylc/flow/scheduler.py

```diff
@@ -404,12 +404,7 @@ async def initialise(self):
         self.workflow_event_handler = WorkflowEventHandler(self.proc_pool)
 
         self.xtrigger_mgr = XtriggerManager(
-            self.workflow,
-            user=self.owner,
-            broadcast_mgr=self.broadcast_mgr,
-            workflow_db_mgr=self.workflow_db_mgr,
-            data_store_mgr=self.data_store_mgr,
-            proc_pool=self.proc_pool,
+            self,
```

Comment:
Task pool isn't instantiated yet, but will be when `xtrigger_mgr` calls it.. (also `xtrigger_mgr` is passed into pool already)